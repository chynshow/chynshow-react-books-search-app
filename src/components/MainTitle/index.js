import React from 'react';

const MainTitle = () => (
  <h3 className='c-main-title'>Which book do you want to read next?</h3>
);

export default MainTitle;
