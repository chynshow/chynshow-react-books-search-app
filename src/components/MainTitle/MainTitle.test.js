import React from 'react';
import MainTitle from './';
import { shallow } from 'enzyme';

describe('MainTitle component', () => {
  it('should render MainTitle component', () => {
    const c = shallow(<MainTitle />);
    expect(c).toMatchSnapshot();
  });
});
