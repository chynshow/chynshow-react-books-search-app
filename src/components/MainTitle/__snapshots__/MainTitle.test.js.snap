// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainTitle component should render MainTitle component 1`] = `
<h3
  className="c-main-title"
>
  Which book do you want to read next?
</h3>
`;
